app/customer.py

```diff
@@ -0,0 +1,24 @@
+from app.car import Car
+
+from dataclasses import dataclass
+import math
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list[int]
+    money: int
+    car: "Car(**car)"
+
+    def calculate_trip_cost(self,
+                            shop_location: list[int],
+                            car: "Car"
+                            ) -> float:

```

Comment:
In case there are no arguments on the same line with an closing parenthesis, your opening parenthesis should not have any arguments on the same line as well.